content/millennium/dstu2/general-clinical/condition.md

```diff
@@ -127,7 +127,13 @@ Create a new condition.
 
 _Implementation Notes_
 
-* Currently, `health-concern` category code is not supported for writing conditions.
+* A few restrictions are in place based on `Condition.category`
+  * For Conditions with a category code of [diagnosis](http://hl7.org/fhir/dstu2/valueset-condition-category.html):
+    * `Condition.encounter` is required
+    * `Condition.abatementDateTime` is not supported
+  * For Conditions with a category code of [problem](https://argonautwiki.hl7.org/Argonaut_Condition_Category_Codes):
+    * If `Condition.dateRecorded` is set on the request body, its value will currently be ignored
+  * Creating Conditions with a category code of [health-concern](https://argonautwiki.hl7.org/Argonaut_Condition_Category_Codes) is not currently supported
```

Comment:
NIT: do we want to end all these sentences with a period?

Reply:
yep: https://github.com/cerner/fhir.cerner.com/pull/458/commits/0f5401901262fb985ecf90168eca78ced7cf4b87